subsys/net/ip/tcp.c

```diff
@@ -28,6 +28,8 @@ LOG_MODULE_REGISTER(net_tcp, CONFIG_NET_TCP_LOG_LEVEL);
 
 #define ACK_TIMEOUT_MS CONFIG_NET_TCP_ACK_TIMEOUT
 #define ACK_TIMEOUT K_MSEC(ACK_TIMEOUT_MS)
+#define LAST_ACK_TIMEOUT_MS tcp_fin_timeout_ms
```

Comment:
Just one question, what is the default value here?

Reply:
Same as for active close, total retransmission timeout, calculated on boot here:
https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/net/ip/tcp.c#L4341